### PR TITLE
c07 translation

### PR DESCRIPTION
**Pull request by ManlyMarco**
Thanks to @ScrewThisNoise for providing the translations.

### CODE REVIEW

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c07/00/0/translation.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-「あはは。お前って面白い奴なんだな！」=“Ahaha. You sure are an interesting guy!”
+「あはは。お前って面白い奴なんだな！」=“Ahaha. You a real interesting guy, you know that?”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c07/00/0/translation.txt:4]: Assuming this a grammar error
```suggestion
「あはは。お前って面白い奴なんだな！」=“Ahaha. You're a real interesting guy, you know that?”
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c07/00/1002/translation.txt [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-「これからは女子相手じゃなくて、向かうところ敵知らずって言うことにするか」=“From now on, instead of never losing against girls, I'm gonna say I've never lost against anyone who challenged me.”
-「ぐぬぬ、もうちょっとなんとかなると思ったのに」=“Mwah, I thought that you'd be a little better.”
-「修行が足りなかったな？　さて、何を言うこと聞かせるか」=“Guess you better practice more eh? Now then, what should I have you do for me?”
-「いつの間に負けた方が言うことを聞くルールになってる……」=“Since when did it become a rule that the winner gets to make a demand...”
-「そうだ！　これからしばらく家来な。私を敬うがいい」=“Oh I know! I'll have you be my manservant for a while. Now praise me!”
-「かしこまりました、[H名]様」=“As you wish, Madame [H名].”
-「ノリノリかよ」=“Wow, you taking this seriously?”
-「[H名]様におかれましては本日もお変わりなく、大変お美しゅうございます」=“May I say that, Madame [H名], you look just as wonderful today as you've ever been.”
-「っ、や、やっぱし罰ゲームなしで！」=“O-OK, no more a punishment game!”
+「[H名]は卓球とか得意なんじゃない？」=“Aren't you pretty good at ping pong, [H]?”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c07/00/1002/translation.txt:4]: There seem to be a lot of these than have changed, which I don't think is correct. May want to do a global check substitutions match on both sides of equals in general (if so, we can handle that after).

```suggestion
「[H名]は卓球とか得意なんじゃない？」=“Aren't you pretty good at ping pong, [H名]?”
```

**Review by GeBo1 — CHANGES_REQUESTED** [review]
Some of these were probably better before, but it's not a big deal. the commend about the `[H]` substitutions applies across the board. They should probably all be updated to match correctly.

**Review by GeBo1 — APPROVED** [review]

File: Translation/en/RedirectedResources/assets/abdata/communication/info_00/communication_23/translation.txt [hunk 3]
```diff
@@ -298,7 +298,7 @@
 「……むむ、ファンとして見に行くべきか」=“...Mumuh, as a fan, guess I've no choice but to go watch it.”
 //「おススメの漫画、教えてよ」=
 //「お安い御用ですぞ。どんなジャンルがいいか言ってくれれば厳選するよ」=
-「好きだよ、[H]。[H]はどう、かな？」=“I love you, [H]。[H]. What, about you?”
+「好きだよ、[H]。[H]はどう、かな？」=“I love you, [H]。[H]。[H]。[H]. What, about you?”
```
Comment (ManlyMarco) [inline Translation/en/RedirectedResources/assets/abdata/communication/info_00/communication_23/translation.txt:301]:
``` suggestion
「好きだよ、[H]。[H]はどう、かな？」=“I love you, [H]. What about you?”
```

**Pull request merged**